sdk/lib/drivers/ip/transport/tcp/tcp.c

```diff
@@ -214,7 +214,7 @@ NTSTATUS TCPStartup(VOID)
 {
     NTSTATUS Status;
 
-    Status = PortsStartup( &TCPPorts, 1, 0xfffe );
+    Status = PortsStartup( &TCPPorts, 1, 0xffff );
```

Comment:
```suggestion
    Status = PortsStartup(&TCPPorts, 1, 0xffff);
```

Reply:
cosmetic, but OK

Reply:
Yes, see discussion with Mark in ticket's comment